presto-main/src/main/java/com/facebook/presto/cost/CostCalculatorUsingExchanges.java

```diff
@@ -259,40 +251,40 @@ public PlanCostEstimate visitSpatialJoin(SpatialJoinNode node, Void context)
         }
 
         @Override
-        public PlanCostEstimate visitValues(ValuesNode node, Void context)
+        public LocalCostEstimate visitValues(ValuesNode node, Void context)
         {
-            return PlanCostEstimate.zero();
+            return LocalCostEstimate.zero();
         }
 
         @Override
-        public PlanCostEstimate visitEnforceSingleRow(EnforceSingleRowNode node, Void context)
+        public LocalCostEstimate visitEnforceSingleRow(EnforceSingleRowNode node, Void context)
         {
-            return PlanCostEstimate.zero();
+            return LocalCostEstimate.zero();
         }
 
         @Override
-        public PlanCostEstimate visitLimit(LimitNode node, Void context)
+        public LocalCostEstimate visitLimit(LimitNode node, Void context)
         {
             // This is just a wild guess. First of all, LimitNode is rather rare except as a top node of a query plan,
             // so proper cost estimation is not that important. Second, since LimitNode can lead to incomplete evaluation
             // of the source, true cost estimation should be implemented as a "constraint" enforced on a sub-tree and
             // evaluated in context of actual source node type (and their sources).
-            return cpuCost(getStats(node).getOutputSizeInBytes(node.getOutputSymbols(), types));
+            return LocalCostEstimate.ofCpu(getStats(node).getOutputSizeInBytes(node.getOutputSymbols(), types));
         }
 
         @Override
-        public PlanCostEstimate visitUnion(UnionNode node, Void context)
+        public LocalCostEstimate visitUnion(UnionNode node, Void context)
         {
             // Cost will be accounted either in CostCalculatorUsingExchanges#CostEstimator#visitExchanged
             // or in CostCalculatorWithEstimatedExchanges#CostEstimator#visitUnion
             // This stub is needed just to avoid the cumulative cost being set to unknown
-            return PlanCostEstimate.zero();
+            return LocalCostEstimate.zero();
         }
 
         @Override
-        public PlanCostEstimate visitSort(SortNode node, Void context)
+        public LocalCostEstimate visitSort(SortNode node, Void context)
```

Comment:
I add this change during backport. Looks like when @findepi ports https://github.com/prestodb/presto/pull/12347 via https://github.com/prestosql/presto/pull/336, `visitSort` is not ported in PrestoSQL's version :) 

cc @picrinite , @rschlussel , @arhimondr